src/sentry/monitors/logic/mark_ok.py

```diff
@@ -43,22 +43,49 @@ def mark_ok(checkin: MonitorCheckIn, ts: datetime):
                 previous_checkin["status"] == CheckInStatus.OK
                 for previous_checkin in previous_checkins
             )
+        else:
+            incident_recovering = True
```

Comment:
Does this mean we're just __always__ resolving issues for monitors that don't use thresholds when an OK check-in comes in now?

Reply:
well we just run an update which can be a no-op

Reply:
also there could be a failure threshold but not a recovery one and we need to cover that

Reply:
i guess previously we weren't closing all incidents correctly if there's a failure threshold but not a recovery one

Reply:
I just mean, the previous behavior was that monitors that do not have thresholds never have their issues resolved right. How it looks like we're actively resolving issues for those monitors no?

Reply:
Let's go ahead and add a comment as to why we're setting this to true